### PR TITLE
[Rule Tuning] Security Software Discovery via Grep

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/993

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 1]
```diff
@@ -1,58 +1,61 @@
...
-          avscan or
-          guard or
-          rtvscand or
-          symcfgd or
-          scmdaemon or
-          symantec or
-          elastic-endpoint
-          )
+process where process.name : "grep" and user.id != "0" and
+ not process.parent.executable: "/Library/Application Support/*" and
```
Comment (brokensound77) [inline rules/cross-platform/discovery_security_software_grep.toml:26]: nit:
```suggestion
 not process.parent.executable : "/Library/Application Support/*" and
```

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
+          "guard*",
+          "rtvscand*",
+          "symcfgd*",
+          "scmdaemon*",
+          "symantec*",
+          "sophos*",
+          "osquery*",
+          "elastic-endpoint*"
+          ) and
+   not (process.args:"Avast" and process.args:"Passwords")
```
Comment (brokensound77) [inline rules/cross-platform/discovery_security_software_grep.toml:58]: nit 
```suggestion
   not (process.args : "Avast" and process.args : "Passwords")
```

**Review by brokensound77 — APPROVED** [review]
LGTM, thanks 👍

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 3]
```diff
@@ -1,59 +1,64 @@
 [metadata]
 creation_date = "2020/12/20"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/03/08"
+
```
Comment (brokensound77) [inline rules/cross-platform/discovery_security_software_grep.toml:5]:
```suggestion
```

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 4]
```diff
@@ -1,59 +1,64 @@
...
 license = "Elastic License v2"
 name = "Security Software Discovery via Grep"
 risk_score = 47
 rule_id = "870aecc0-cea4-4110-af3f-e02e9b373655"
 severity = "medium"
 tags = ["Elastic", "Host", "macOS", "Linux", "Threat Detection", "Discovery"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
+
```
Comment (brokensound77) [inline rules/cross-platform/discovery_security_software_grep.toml:25]:
```suggestion
```

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 5]
```diff
@@ -1,59 +1,62 @@
...
-updated_date = "2021/03/03"
+updated_date = "2021/03/08"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies the use of the grep command to discover known third-party macOS and Linux security tools, such as Antivirus
 or Host Firewall details.
 """
+false_positives = ["Endpoint Security Installers, Updaters and Post Installation verification scripts."]
```
Comment (bm11100) [inline rules/cross-platform/discovery_security_software_grep.toml:12]:
```suggestion
false_positives = ["Endpoint Security installers, updaters and post installation verification scripts."]
```

File: rules/cross-platform/discovery_security_software_grep.toml [hunk 6]
```diff
@@ -1,59 +1,62 @@
...
-          kesl or
-          avscan or
-          guard or
-          rtvscand or
-          symcfgd or
-          scmdaemon or
-          symantec or
-          elastic-endpoint
-          )
+process where process.name : "grep" and user.id != "0" and
```
Comment (bm11100) [inline rules/cross-platform/discovery_security_software_grep.toml:26]:
```suggestion
process where event.type == "start" and
process.name : "grep" and user.id != "0" and
```

**Review by bm11100 — APPROVED** [review]

**Pull request merged**